README.md

```diff
@@ -54,10 +54,33 @@ module.exports = {
 
 ### ./.eslintrc
 
+#### Use all rules as errors
+
+```json
+{
+  "plugins": ["local-rules"],
+  "extends": [
+    "plugin:local-rules/error"
+  ]
+}
+```
```

Comment:
I added this on top because I think that it's the most common usage and it has the least amount of config